qt/widgets/common/src/PropertyHandler.cpp

```diff
@@ -1379,6 +1379,28 @@ double PropertyHandler::fwhm() const {
   return 0;
 }
 
+std::string PropertyHandler::getWidthParameterName() const {
+  if (m_pf) {
+    return m_pf->getWidthParameterName();
+  }
+  return std::string();
```

Comment:
As above we can return ""
```suggestion
  return ""
```